### DISCUSSION TITLE
My approach to building MiniDexed

### DISCUSSION THREAD

**Discussion by Picard-Synth** [probonopd/MiniDexed#692] [Show and tell]
This is my approach based on a Raspi 3 toghether with a dedicated home brew "all-in-one" board that integrates the 5102 DAC, encoder, 9 buttons (Cherry ML type), 5pin-MIDI, 2x1/4" line out jacks, headphone amplifier + jack, and 2 volume pots. Case is 3d printed, painted, and labelled.

![MDa](https://github.com/user-attachments/assets/134ea979-fd97-4710-966f-2cc807b32f9d)

Volume is adjusted separately for headphone and line, which is especially useful in jam situations where you can pre-listen and eventually kick in.
Also, I used a locking, durable power connector (miniXLR) instead of microUSB which isn't really stage-ready...

![MDb](https://github.com/user-attachments/assets/9fb9996b-fa9c-474d-a941-b58b4b4133a4)

PCB is SMT with a self-imposed size limit of 100x100 mm.

![Minidexed3_ray](https://github.com/user-attachments/assets/93397030-171d-4f6e-8d38-c398c49fcc77)
![Minidexed3_edit](https://github.com/user-attachments/assets/0ec77b10-3833-4c8e-be64-e96d7a0a1736)

**Comment by Banana71** [comment]
I'm really impressed. Very nice and functional version of a miniDexed. 
But where is the micro SD card?
How did you build the circuit for the volume control for the headphone line out?

**Reply by Picard-Synth** [reply]
Thanks a lot.
Yeah - SD card access is only by opening the enclosure. For me it's rather acceptable.
I used the TPA6138A headphone amplifier IC and I'm quite satisfied how good it works at 3.3V only. Since it uses an internal charge pump for its negative supply headphones can be connected without coupling capacitors. I use a voltage gain of 1.5x to max out the 5102 output of 2.1Vpp to 3.1Vpp and the 6138 outputs it as +/-3.1V ... nice!
Pots are dual 50k audio.
Wiring is like this:
![MD_hp](https://github.com/user-attachments/assets/743d6a42-0038-4f00-a5d3-ec46a7b32fe2)

**Comment by Onosendaifr** [comment]
Absolutly beautifull !

**Reply by Picard-Synth** [reply]
Thanks man!

**Reply by Onosendaifr** [reply]
Do you have an email ? I have a question for you

**Reply by Picard-Synth** [reply]
Hi Onosendaifr, you can send me a private message here at [modwiggler](https://www.modwiggler.com/forum/viewtopic.php?p=4204133#p4204133).

**Comment by diyelectromusic** [comment]
That looks great.  And I really like the graphics on the case.  We'd better not change the order of the menus now then! :)

Kevin

**Reply by Picard-Synth** [reply]
Oh yes please not!

**Comment by Picard-Synth** [comment]
It was also my first Tinkercad project for the enclosure:

![MD_encl](https://github.com/user-attachments/assets/e98557af-1a52-4aa7-ad12-a2239aac92f9)
![MD_encl2](https://github.com/user-attachments/assets/3a35ecce-6438-4746-9f70-09c768eebcb7)

**Comment by blixabloxa** [comment]
That is a very professional looking job. I love it!

**Comment by Kirtai** [comment]
That is *gorgeous*.
